module/solutions/helper.js

```diff
@@ -127,4 +127,24 @@ module.exports = class solutionsHelper {
 
     }
 
+    static generateSolutionsExternalId(solutionId){
```

Comment:
remove this one and use solutionDocument function in solutionHelper

Reply:
Resolved